src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+
+interface FilterProps {
+  currentFilter: string;
+  onSelectFilter: (filterType: string) => void;
+}
+
+export const Filter: React.FC<FilterProps> = ({
+  currentFilter,
+  onSelectFilter,
+}) => {
+  const filterTypes = ['all', 'active', 'completed'];
+
+  const handleSelectFilter = (filterType: string) => {
+    if (currentFilter !== filterType && filterTypes.includes(filterType)) {
+      onSelectFilter(filterType);
+    }
+  };
+
+  return (
+    <nav className="filter">
+      {filterTypes.map((filter) => {
+        let title: string | string[] = filter.split('');
+
+        title[0] = title[0].toUpperCase();
+
+        title = title.join('');
```

Comment:
If the goal of this is to uppercase the first letter you can do it like this

`const title = filter.charAt(0).toUpperCase() + filter.slice(1);`

Reply:
It is, thank you